index.html

```diff
@@ -0,0 +1,1376 @@
+                            <a href="#" class="feature-hover"> Video Shows ICE Agents Smash Car Window, Drag Father Out
+                                in
+                                Front
+                                of Family</a>
+                        </h6>
+                        <p class="mt-2 featured-stories-subheadline">
+                            The family was on their way to a doctor's appointment for their daughter when their vehicle
+                            was
+                            blocked in by U.S. Immigration and Customs Enforcement agents.
+                        </p>
+                    </article>
+                    <!--/featured stories 3-->
+
+                    <!--featured stories 4-->
+
+                    <article>
+                        <div class="wrap-cover">
+                            <div class="wrap">
+                                <a href="#">Editor's Notes</a>
+                            </div>
+
+                            <div><a href="#"><img src="assets/featured-4.png" alt="featured"
+                                        class="feature img-fluid"></a>
+                            </div>
+
+                        </div>
+                        <h6 class="mt-2 font-weight-bold featured-stories-mainheadline">
+                            <a href="#" class="feature-hover"> To Our Readers:</a>
+                        </h6>
+                        <p class="mt-2 featured-stories-subheadline">
+                            We always want to hear from you, and to make that easier, we've implemented a new commenting
+                            forum.
+                            Take a look at the bottom of Newsweek articles. Spot.IM is a tool that helps readers respond
+                            to
+                            our
+                            stories and communicate with one another—and with us. Ask us a question, embed a GIF, point
+                            out
+                            a
+                            mistake. Be civil and respectful, please. And most important: Be in touch.
+                        </p>
+                    </article>
+                    <!--/featured stories 4-->
+
+                </section>
+                <!--END OF FS SECTION-->
+
+                <!--START OF TOP-STORY SECTION-->
+                <section class="col-lg-5 col-xl-5 col-sm-12 mt-4 col-md-7 order-sm-first order-md-second">
+                    <h6 class="font-weight-bold section-heading">
+                        TOP STORY
+                    </h6>
+
+                    <!--top story: Main Image-->
+                    <section>
+                        <article>
+                            <div class="wrap-cover">
+                                <div class="wrap">
+                                    <a href="#">u.s</a>
+                                </div>
+                                <div>
```

Comment:
Same idea for more stories section, wrapped around `<section>` and each story has `<article>` tag.